include/private/gc_priv.h

```diff
@@ -857,7 +857,9 @@ EXTERN_C_END
 # endif
 #endif /* DARWIN */
 
-#include <setjmp.h>
+#ifndef GC_NO_SIGSETJMP
```

Comment:
It looks like my hypothesis that using this macro here would be safe was invalid

https://ci.appveyor.com/project/ivmai/bdwgc/builds/45688100/job/72kkvn3k4tfdwxro

Not too sure what would be a good way to handle this missing header file though, appreciate any pointer